_guidelines-v4/04-cmn/03-cmnadditionalmeasurecontent/02-cmnslurties.md

```diff
@@ -25,37 +25,37 @@ This attribute allows three values:
 The scope of the **@tie** attribute is the musical {% include link elem="layer" %}; that is, a tie started in one layer may only be ended by a subsequent musical event with a **@tie** attribute with an *m* or *t* value in the same layer. The tie-terminating event may lie in the following measure.
 
 {% include figure img="modules/cmn/finger-300.png" caption="Ties across barlines" %}
-{% include mei example="cmn/cmn-sample115.xml" valid="" %}
+{% include mei example="cmn/cmn-sample115.txt" valid="" %}
 
 When **@tie** is used on chords, it functions as a shorthand indication for multiple tie markings; that is, a separate tie is drawn for every pitch in the chord that remains unchanged in the succeeding chord.
 
-{% include mei example="cmn/cmn-sample116.xml" valid="" %}
+{% include mei example="cmn/cmn-sample116.txt" valid="" %}
 
 This is equivalent to the following, more verbose version:
 
-{% include mei example="cmn/cmn-sample117.xml" valid="" %}
+{% include mei example="cmn/cmn-sample117.txt" valid="" %}
 
 A slur is a curved line that connects a group of notes of different pitch. It normally indicates that an instrument-specific performance technique should be applied to the affected notes. For example, in notation for winds, the notes should be played in one breath, while a single bow is indicated for string instruments.
 
 {% include figure img="ExampleImages/slur-300-20100514.png" caption="Slurs" %}
 
 In MEI, slurs may be encoded in a similar way to ties: {% include link elem="note" %} and {% include link elem="chord" %} bear a **@slur** attribute that allows the commencement or ending of a slur at this element. The allowed values, however, are slightly different: The *i*, *m* or *t* are followed by a single digit in the range 1 to 6, as in the following example:
```

Comment:
As I would generally recommend to use the `<slur>` element over the `@slur` attribute, I'm not sure about the order here. Yes, for ties there are more arguments to encode them as attribute, so that seems ok. However, introducing them as elements first seems equally fine for me. I wonder if we shouldn't turn this round, describe the resp. elements first, and then introduce the attributes as a solution that mostly addresses analytical purposes (which isn't our focus elsewhere)?